src/sqerl_client.erl

```diff
@@ -257,3 +246,30 @@ read_statements_from_config() ->
             error_logger:error_report(Msg),
             error(Msg)
     end.
+
+
+%% Helper function to report and error
+
+log_and_error(Msg) ->
+    error_logger:error_report(Msg),
+    error(Msg).
+
+maybe_log(true, QueryOrName, Args) ->
+    error_logger:info_msg("(~p) Executing statement ~p with args ~p~n", [self(), QueryOrName, Args]);
```

Comment:
Let's consider a more structured log format... (perhaps for discussion and separate PR)